.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+daysUntilClose: false
```

Comment:
Good question

Reply:
It might not be that good if I'm not mistaken, it might be based on a now deprecated application. I've created https://github.com/openzim/_python-bootstrap/issues/34 to discuss this, I think we can merge with current configuration until upstream issue has settled.